apis/clusters/v1beta1/cadence_types.go

```diff
@@ -305,6 +313,17 @@ func (cd *CadenceDataCentre) privateLinkToInstAPI() (iPrivateLink []*models.Priv
 	return
 }
 
+func (cs *CadenceSpec) TargetCadenceToInstAPI() []*models.TargetCadence {
+	var targets []*models.TargetCadence
+	for _, target := range cs.TargetPrimaryCadence {
+		targets = append(targets, &models.TargetCadence{
+			DependencyCDCID:   target.DependencyCDCID,
+			DependencyVPCType: target.DependencyVPCType,
+		})
+	}
```

Comment:
Add a new line here and fix similar cases below please

Reply:
Fixed.